Source/Common/Config.cpp

```diff
@@ -221,12 +221,18 @@ namespace JSON {
         continue;
 
       std::string_view Key = Var.substr(0,pos);
-      std::string_view Value {Var.substr(pos+1)};
+      std::string_view Value_View {Var.substr(pos+1)};
+      std::optional<fextl::string> Value;
 
 #define ENVLOADER
 #include <FEXCore/Config/ConfigOptions.inl>
 
-      EnvMap[Key] = Value;
+      if (Value) {
+        EnvMap[Key] = *Value;
```

Comment:
should these be using insert_or_assign, like the TestEnvLoader below does?

Reply:
switched over to insert_or_assign.